Gemfile

```diff
@@ -8,10 +8,6 @@ group :test do
   gem "codeclimate-test-reporter", "~> 1.0.0", :require => false
 end
 
-unless dependencies.detect { |d| d.name == "xclarity_client" }
-  gem "xclarity_client"
-end
-
```

Comment:
maybe it was there in the first iteration?
I'm pretty sure it can go away as it's already in the gemspec